db/pre-re/status.yml

```diff
@@ -231,6 +236,7 @@ Body:
       Inspiration: true
       Fear: true
       Gvg_Blind: true
+      Blind: true
   - Status: Bleeding
     Icon: EFST_BLOODING
     DurationLookup: NPC_BLEEDING
```

Comment:
Currently Bleeding is blocking itself because of the "OverlapFail" flag. It should not. If get inflicted by Bleeding while you still have Bleeding, it will reset the timer.

Please also check Whiteimprison, Confusion, DPoison and Fear, those can also overlap!